src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar Thanks for the testing :+1: 
That a good new :smile: i was sure that it was sustainable in some way.
So with hardware acceleration your 200 Mb/s :flushed: should be fine and certainly 400 Mb/s as well :wink: 

I'll make new commit in this PR and ask the user to compare these 2 command lines, because with you using FFmpeg64 and FFmpeg i've difficulty to compare:
```
ffmpeg64.exe -y -hwaccel auto -i jellyfish-120-mbps-4k-uhd-h264.mkv -c:v h264_nvenc -tune fastdecode -crf 22 -c:a aac -b:a
 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark
```
```
ffmpeg64.exe -y -hwaccel auto -i jellyfish-120-mbps-4k-uhd-h264.mkv -c:v h264_nvenc -tune fastdecode -crf 22 -c:a aac -b:a
 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark
ffmpeg64.exe -y -hwaccel auto -i jellyfish-120-mbps-4k-uhd-h264.mkv -c:v h264_nvenc -tune fastdecode -crf 22 -c:a aac -b:a
 320k -ar 48000 -pix_fmt yuv420p -f mp4 -movflags frag_keyframe+empty_moov -seekable 1  hevc_test1.mp4 -benchmark
```

It should be nice to compare the different accel savors, at least with the ones that you have:
```
h264_cuda
h264_cuvid
h264_d3d11va
h264_dxva2
h264_libmfx
```